Sources/DarkModeCore/UIColor+DarkModeKit.h

```diff
@@ -4,7 +4,11 @@
 //
 
 #import <UIKit/UIKit.h>
+#ifdef SWIFT_PACKAGE
+#import "DMNamespace.h"
```

Comment:
I'm wondering can we use `#import ""` instead `#import <>` for Xcode framework too. I'm not sure will it lead problem or not.

Reply:
that is probably a no, I think Xcode framework is pretty much like Carthage, where you only get a .framework bundle with headers and binary. While public UIKit headers use `<>`, we should also be using it instead of `""`